layouts/partials/embedshare.haml

```diff
@@ -0,0 +1,45 @@
+%h3 Embed into your Website
+%input{:type => "text", :size=> "85", :readonly => true, 
+       :value => "<iframe width=\"853\" height=\"480\" src=\"#{oembed_page_url(@item.identifier)}\" frameborder=\"0\" allowfullscreen></iframe>"}
+%h3 Share with others
+%ul
+  %li
+    %a{:id => "btnTweet", :href => twitter_url(@item[:title], page_url(@item)) }
+      via Twitter
+  %li
+    %a{:id => "btnFacebook",
+       :href => facebook_url(@item[:title], page_url(@item)) }
+      via Facebook
+  %li
+    %a{:id => "btnGPlus", :href => googleplus_url(@item[:title], page_url(@item)) }
+      via Google+
+  %li
+    %a{:id => "btnAppNet", :href => appnet_url(@item[:title], page_url(@item)) }
+      via App.net
+  %li
+    %a{:id => "btnMail", :href => mail_url(@item[:title], page_url(@item)) }
+      by Mail
+:javascript
+  $(function () {
+    $('#btnTweet').click(function (e) {
+      window.open(this.href, 'tweet it', 'width=550,height=420,resizable=yes');
+      e.preventDefault();
+    });
+    $('#btnFacebook').click(function (e) {
+      window.open(this.href, 'share it', 'width=550,height=420,resizable=yes');
+      e.preventDefault();
+    });
+    $('#btnGPlus').click(function (e) {
+      window.open(this.href, 'plus it', 'width=550,height=420,resizable=yes');
+      e.preventDefault();
+    });
+    $('#btnAppNet').click(function (e) {
+      window.open(this.href, 'share it', 'width=550,height=420,resizable=yes');
+      e.preventDefault();
+    });
+    $('#btnMail').click(function (e) {
```

Comment:
I'm unsure if opening a mailto-link in a popup would not leave a stray empty popup open (it does at least in my chromium). I'd suggest to remove the js-click-handler on the mailto:-Link altogether.